src/main/webapp/js/richTextEditor.js

```diff
@@ -37,11 +37,10 @@ var richTextEditorBuilder = {
         };
     },
 
-    initEditor: function(selector, inline) {
+    initEditor: function(selector, opts) {
         tinymce.init(jQuery.extend(this.getDefaultConfiguration(), {
```

Comment:
`$.each`, `$.ajax` are commonly used. Pretty sure there are `$.extend` used as well. No policy on `jQuery` or `$` written on stone, but let's make it more uniform by changing it to `$`.